web/yo/app/scripts/directives/reviewpanel.js

```diff
@@ -232,6 +243,26 @@ angular.module('oncokbApp')
                             $scope.tm.trials.pushAll($scope.tm.trials_review.get('lastReviewed'));
                             rejectItem([{reviewObj: $scope.tm.trials_review, uuid: $scope.tm.trials_uuid}]);
                             break;
+                        case 'MUTATION_NAME_CHANGE':
+                            rejectItem([{obj: $scope.mt.name, reviewObj: $scope.mt.name_review, uuid: $scope.mt.name_uuid}]);
+                            break;
+                        case 'TUMOR_NAME_CHANGE':
+                            var lastReviewed = $rootScope.model.createList();
+                            _.each($scope.tm.cancerTypes_review.get('lastReviewed'), function(ct) {
+                                var cancerType = $rootScope.model.create(OncoKB.CancerType);
```

Comment:
It is not necessary. Since app.js is included, and OncoKB is a global variable outside of any angular module, we can use it directly.